[2022-05-02 16:56:50,619] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: podsight_ETL.extract_podsight_data manual__2022-05-02T16:50:43+00:00 [queued]>
[2022-05-02 16:56:50,629] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: podsight_ETL.extract_podsight_data manual__2022-05-02T16:50:43+00:00 [queued]>
[2022-05-02 16:56:50,630] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 16:56:50,630] {taskinstance.py:1250} INFO - Starting attempt 4 of 6
[2022-05-02 16:56:50,631] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-05-02 16:56:50,643] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): extract_podsight_data> on 2022-05-02 16:50:43+00:00
[2022-05-02 16:56:50,647] {standard_task_runner.py:52} INFO - Started process 592 to run task
[2022-05-02 16:56:50,651] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'podsight_ETL', 'extract_podsight_data', 'manual__2022-05-02T16:50:43+00:00', '--job-id', '110', '--raw', '--subdir', 'DAGS_FOLDER/podsight_ETL.py', '--cfg-path', '/tmp/tmp0pblmat6', '--error-file', '/tmp/tmpkow1bevg']
[2022-05-02 16:56:50,652] {standard_task_runner.py:80} INFO - Job 110: Subtask extract_podsight_data
[2022-05-02 16:56:50,709] {logging_mixin.py:109} INFO - Running <TaskInstance: podsight_ETL.extract_podsight_data manual__2022-05-02T16:50:43+00:00 [running]> on host 4c9d01f48299
[2022-05-02 16:56:50,763] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=hamza.ahmed@liquidagency.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=podsight_ETL
AIRFLOW_CTX_TASK_ID=extract_podsight_data
AIRFLOW_CTX_EXECUTION_DATE=2022-05-02T16:50:43+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-02T16:50:43+00:00
[2022-05-02 16:56:50,765] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-02 16:56:50,767] {subprocess.py:74} INFO - Running command: ['bash', '-c', '/Users/hamza.ahmed/Coding Projects/Python_Apps/Docker_Applications/Airflow_Docker_Apps/Python_Scripts/podsight_API_JumpCloud.py']
[2022-05-02 16:56:50,778] {subprocess.py:85} INFO - Output:
[2022-05-02 16:56:50,780] {subprocess.py:89} INFO - bash: /Users/hamza.ahmed/Coding: No such file or directory
[2022-05-02 16:56:50,781] {subprocess.py:93} INFO - Command exited with return code 127
[2022-05-02 16:56:50,797] {taskinstance.py:1774} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 188, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-05-02 16:56:50,807] {taskinstance.py:1288} INFO - Marking task as UP_FOR_RETRY. dag_id=podsight_ETL, task_id=extract_podsight_data, execution_date=20220502T165043, start_date=20220502T165650, end_date=20220502T165650
[2022-05-02 16:56:50,819] {standard_task_runner.py:98} ERROR - Failed to execute job 110 for task extract_podsight_data (Bash command failed. The command returned a non-zero exit code 127.; 592)
[2022-05-02 16:56:50,865] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-05-02 16:56:50,898] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
